src/core/Akka.Persistence/Journal/AsyncWriteJournal.cs

```diff
@@ -161,17 +163,17 @@ protected AsyncWriteJournal()
         /// This call is protected with a circuit-breaker.
         /// </summary>
         /// <param name="messages">TBD</param>
-        /// <returns>TBD</returns>
-        protected abstract Task<IImmutableList<Exception>> WriteMessagesAsync(IEnumerable<AtomicWrite> messages);
+        /// <param name="cancellationToken"><see cref="CancellationToken"/> used to signal cancelled snapshot operation</param> 
+        protected abstract Task<IImmutableList<Exception>> WriteMessagesAsync(IEnumerable<AtomicWrite> messages, CancellationToken cancellationToken);
```

Comment:
[Breaking change] `Akka.Persistence` codes that are being wrapped inside a `CircuitBreaker` will now take a `CancellationToken` as a signal that the operation has been timed out by the `CircuitBreaker`.

Reply:
LGTM